src/lib/collab/stack-client.js

```diff
@@ -383,5 +384,25 @@ export class ServiceClient {
       telepointerDoc
     )
   }
+
+  async postImage(name, noteId, file) {
+    return this.stackClient.fetchJSON(
+      'POST',
+      this.path(noteId, `images?Name=${name}`),
+      file,
+      { headers: { 'Content-Type': 'image/jpeg' } }
+    )
+  }
+
+  async getImages(noteId) {
+    return this.stackClient.fetchJSON(
+      'GET',
+      `/data/${appConstants.GET_IMAGES_DOCTYPE}/${noteId}/relationships/references`
+    )
```

Comment:
We should not call the stackClient directly. Is there a way trough client.getById(doctype,id) to fetch relation and reference?

Reply:
Sorry not sure to understand but basically I implemented it like other endpoints were implemented, not thinking much about it except staying consistent, but I can look into that

Reply:
Fixed